grpc/src/main/java/io/stargate/grpc/impl/GrpcImpl.java

```diff
@@ -72,6 +84,13 @@ public void start() {
   public void stop() {
     try {
       server.shutdown().awaitTermination();
+      // Since we provided our own executor, it's our responsibility to shut it down.
+      // Note that we don't handle restarts because GrpcActivator never reuses an existing instance
+      // (and that wouldn't work anyway, because Server doesn't support it either).
+      executor.shutdown();
+      if (!executor.awaitTermination(1, TimeUnit.MINUTES)) {
```

Comment:
the `executor.awaitTermination(1, TimeUnit.MINUT)` is blocking. 
It seems that in the worst case, it may slow down the shutdown of Stargate by one minute. Isn't it too much?

Reply:
I can always make it customizable with yet another system property...

We'll also need to address the fact that the previous line calls `server.shutdown().awaitTermination();` which can block indefinitely.